packages/scripts/src/monitor-dvm/MonitorMints.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from "@uma/financial-templates-lib";
+import { ZERO_ADDRESS } from "@uma/common";
+import { VotingTokenEthers } from "@uma/contracts-node";
+import { utils } from "ethers";
+import { logMint } from "./MonitorLogger";
+import { getContractInstanceByUrl } from "../utils/contracts";
+import type { MonitoringParams } from "./common";
+
+export async function monitorMints(logger: typeof Logger, params: MonitoringParams): Promise<void> {
+  const mintsThreshold = utils.parseEther(process.env.MINTS_THRESHOLD || "0");
```

Comment:
i would either turn this into a parameter or add it to monitoring params, same with all the other envs

Reply:
fixed in 228982dc457e606217a9faca1abc24c0e5d5ff14